ibis/expr/operations/sortkeys.py

```diff
@@ -36,19 +36,13 @@ class SortKey(Value):
     shape = rlz.shape_like("expr")
 
     @classmethod
-    def __coerce__(cls, value, T=None, S=None):
-        if isinstance(value, tuple):
-            key, asc = value
-        else:
-            key, asc = value, True
-
-        asc = _is_ascending[asc]
```

Comment:
If this is removed, should we remove also the `_is_ascending` dictionary in lines 13-22?

Reply:
Yup, thanks!

Reply:
Done!